src/MahApps.Metro/MahApps.Metro/Themes/MetroWindow.xaml

```diff
@@ -567,12 +568,12 @@
         <Setter Property="Behaviours:StylizedBehaviors.Behaviors">
             <Setter.Value>
                 <Behaviours:StylizedBehaviorCollection>
-                    <Behaviours:BorderlessWindowBehavior />
                     <Behaviours:WindowsSettingBehaviour />
                     <Behaviours:GlowWindowBehavior />
                 </Behaviours:StylizedBehaviorCollection>
             </Setter.Value>
         </Setter>
```

Comment:
setting the behaviors must be done in `InitializeStylizedBehaviors` at code behind